packages/host/app/services/operator-mode-state-service.ts

```diff
@@ -172,27 +175,32 @@ export default class OperatorModeStateService extends Service {
     }
   }
 
-  async deleteCard(card: CardDef) {
+  async deleteCard(cardId: string) {
```

Comment:
the parameter changed from `CardDef` to a `cardId` because we want to support deleting a card that has an error, but in this case it is not possible to load the card so we only have its `id`. Card's id is the lowest common denonimator of a loaded card and the card's error response so we use that.